app/views/shows/index.html.haml

```diff
@@ -1,3 +1,12 @@
-- @shows.each do |show|
-	%h1= show["title"]
-	%h3= show["type"]
+%h3 Smartflix 
+%h6 Find out more about your favorite movies.
+.row.row-cols-md-6.row-cols-sm-4
```

Comment:
The indentation below 👇 looks wild! It could get worse as you add more contents later on

Reply:
I'm not sure why it's indented like this here, whereas in my machine it looks fine

Reply:
Try indenting with *spaces* rather than *tabs*?

Reply:
Fixed it, though even indenting with spaces failed to fix the issue at first. I had to remove all indentations, commit, and then indent with spaces.